### PR TITLE
Adding FloatAction to animate any property over time.

### PR DESCRIPTION
**Pull request by milos1290**
This commits adds new FloatAction that allows caller to animate any value or property over time. Also includes test case.

### CODE REVIEW

File: cocos/2d/CCActionInterval.h [hunk 1]
```diff
@@ -1525,6 +1525,54 @@ class CC_DLL TargetedAction : public ActionInterval
     CC_DISALLOW_COPY_AND_ASSIGN(TargetedAction);
 };
 
+/**
+ * @class FloatAction
+ * @brief Action used to animate any value in range [from,to] over specified time interval
+ */
+class CC_DLL FloatAction : public ActionInterval
```
Comment (minggo) [inline cocos/2d/CCActionInterval.h]: Some questions:
- FloatAction -> ActionFloat?

File: cocos/2d/CCActionInterval.h [hunk 2]
```diff
@@ -1525,6 +1525,54 @@ class CC_DLL TargetedAction : public ActionInterval
...
+     *  Callback function used to report back result
+     */
+    typedef std::function<void(float value)> FloatActionCallback;
+
+    /**
+     * Creates FloatAction with specified duration, from value, to value and callback to report back
+     * results
+     * @param duration
+     * @param from value
+     * @param to value
```
Comment (minggo) [inline cocos/2d/CCActionInterval.h]: Could you please complete comments for these parameters?

**Comment by milos1290** [comment]
I was away for a week. I'll complete comments and change the name of the action.

**Comment by minggo** [comment]
Thanks.

**Pull request merged**